src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -271,9 +271,13 @@
     <InternalsVisibleTo Include="Microsoft.VisualStudio.ProjectSystem.Managed" />
     <InternalsVisibleTo Include="Microsoft.VisualStudio.ProjectSystem.Managed.VS" />
     <InternalsVisibleTo Include="Microsoft.VisualStudio.ProjectSystem.CSharp" />
+    <InternalsVisibleTo Include="Microsoft.VisualStudio.ProjectSystem.CSharp.VS" />
```

Comment:
This is not a new assembly. This is an [existing assembly](https://github.com/dotnet/project-system/tree/master/src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS).